cdci_osa_plugin/osa_dataserve_dispatcher.py

```diff
@@ -692,7 +692,7 @@ def check_query_roles(self, provided_roles: List[str], par_dic: dict):
         if max_pointings > 50 or len(scw_list) > 50:
             needed_roles.append('unige-hpc-full')            
 
-        if max_pointings > 500 or len(scw_list.split(",")) > 500:
+        if max_pointings > 500 or len(scw_list) > 500:
```

Comment:
is it not done here? https://github.com/oda-hub/dispatcher-plugin-integral/pull/27/files#

Reply:
You're right. I just missed one pull. Sorry I will close and delete.